src/mlnet/Program.cs

```diff
@@ -101,7 +101,7 @@ public static void Main(string[] args)
             }
 
             parser.InvokeAsync(parseResult).Wait();
-            Environment.Exit(0);
+            Environment.Exit(exitCode);
```

Comment:
I think this will return a proper error code in most cases. We'll still miss any errors that we don't catch.

Reply:
Why not just `return exitCode;`? That is the more normal/natural way to return an exit code.

ex. https://github.com/dotnet/cli/blob/21cda4adb6d88d240978e24fcfeb3ebe73f5fdd7/src/dotnet/Program.cs#L30-L83